cli/cmd/root.go

```diff
@@ -7,14 +7,23 @@
 package cmd
 
 import (
+	"context"
+	"os"
+
+	"github.com/edgelesssys/marblerun/cli/internal/cmd"
 	"github.com/spf13/cobra"
 )
 
 // Execute starts the CLI.
 func Execute() error {
-	return NewRootCmd().Execute()
+	cobra.EnableCommandSorting = false
+	rootCmd := NewRootCmd()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	return rootCmd.ExecuteContext(ctx)
```

Comment:
I copied this mostly from our Constellation code, where we use the cancel for "soft" interrupts.
Since we do not use that here, we can probably just use `context.Background()`